target/linux/lantiq/patches-5.10/0801-remoteproc-add-avm_wasp-driver-kernel-module.patch

```diff
@@ -0,0 +1,1294 @@
+--- a/drivers/remoteproc/avm_wasp.c
```

Comment:
Do you have plans to send this patch upstream? Maintaining downstream code is problematic.

Reply:
Yes, took a long time, I am aware from the old discussions of Andreas Böhlers PR for 3390/3490 that it needs to be upstreamed. I don't know how long that takes and if the weird process of using gpio to send the network boot firmware and then send the openwrt image with raw ethernet frames is upstreamable at all.

Reply:
The merge window is open. You can send this patch upstream and see what maintainers think.